Storage Provider Incubation Center/README.md

```diff
@@ -70,12 +70,6 @@
  Storage providers who joined this program can just run with [venus-sealer](https://github.com/filecoin-project/venus-sealer), [venus-wallet](https://github.com/filecoin-project/venus-wallet) and make sure to connect with a distributed mining pool then can run with one or a few nodes at the same time. it will be easier to build your own node on the largest distributed storage network. Happy Mining!
 
 
- ##### As a storage provider of Incubation Center:
-
- Candidates should start submitting applications 2 weeks before each milestone, and the application submission period is 1 week.
-
- Our team will start to review the application to select the storage providers to join this program, and the review period is also 1 week.
-
  #### Venus Master Fellowship Program (Track 2)
```

Comment:
##### Applications window and timelines

Candidates can start submitting applications 2 weeks before the program start date. The application window will be open for one week. After the applications have been submitted, there will be a one week selection phase during which the incubation team will reach out to the applicants and select the winning parties.

Reply:
Due to the need for more communication time and more preparation time for storage providers, it is necessary to extend the application cycle and review cycle. Even review the application form immediately after receiving it without waiting for other participants.

Reply:
Hmm, still cannot be merged.
@jennijuju Can you help on this? 
Thank you.